pyproject.toml

```diff
@@ -85,7 +85,6 @@ source = ["/src/"]
 
 [tool.pytest.ini_options]
 addopts = [
-    '--cov-fail-under=98',
```

Comment:
Does this need to be `'--cov-fail-under=100'` ?

Reply:
Good question, I'm not sure. I think if we specify that it will work. I assumed it would default to that. I'll put it back in as 100.

Reply:
Okay looks good. This is ready to merge now.

Reply:
This doesn't work for me without specifying the `--cov` arg to call the `pytest-cov` extension. I think this needs a revision.

Reply:
Also the current coverage is under 100% so if it did work then no other PR would be able to be merged util 100% coverage is achieved.
Besides this 100% coverage is very extreme for a failing threshold, I'd use something like 95% as a failing rule and make it the onus on the reviewers to make sure appropriate unit testing has been applied to the new features.

Reply:
This MR lifted the coverage to 100% for me, so it should clear that hurdle going forward. I'm happy to bring the tolerance back to 98, but I am also not uncomfortable with 100. Happy to just take the wisdom of the team on this one.